x-pack/plugin/core/src/main/config/log4j2.properties

```diff
@@ -104,6 +104,22 @@ appender.audit_rolling.policies.size.size = 1GB
 appender.audit_rolling.strategy.type = DefaultRolloverStrategy
 appender.audit_rolling.strategy.fileIndex = nomax
 
+appender.behavioral_analytics_rolling.type = RollingFile
```

Comment:
ℹ️ A new logger is added to handle Analytics. The appender is just rendering the message string as received. We expect it to be called with the JSON event.